### ISSUE TITLE
Is this repo in active development?

### ISSUE THREAD

**Issue by finalquest** [jeanregisser/react-native-slider#135]
Hi. 
First off all. Awesome repo.
I should ask.
Is this repo under active development?.
I need to know if should I fork, or summit a pullrequest.

Regards

**Comment by ymahnovskyy** [comment]
Yeah, or maybe someone is supporting this library in some fork? There are a lot of interesting pull requests is there some fork which try to merge at least some pull requests?

**Comment by karltaylor** [comment]
Seems abandoned, @jeanregisser are you still actively developing? Would you consider contributors?

**Comment by TrillCyborg** [comment]
If you want the RTL fixes from the latest release I have published to NPM since @jeanregisser seems to have forgotten to...

@jeanregisser please publish when you can!

https://www.npmjs.com/package/trill-react-native-slider

**Comment by miblanchard** [comment]
I have forked and published a new version that has been updated.  I intend to maintain this and add new features over time - https://www.npmjs.com/package/@miblanchard/react-native-slider

For now I have added multiple thumbs, clicking/tapping, and custom component thumbs.  I am interested in adding a custom component above the thumb while it is moving (think a video thumbnail for a scrubber), and would like to try using `react-native-gesture-handler` and `reanimated` for improved performance.

Please feel free to submit PRs for any of the above!!

**Comment by flyskywhy** [comment]
@miblanchard , I have added react-native-gesture-handler in my forked https://github.com/flyskywhy/react-native-smooth-slider , and it works well in https://github.com/flyskywhy/react-native-slider-color-picker .

Maybe you can ref to the commit which just added react-native-gesture-handler :
https://github.com/flyskywhy/react-native-smooth-slider/commit/59fa00f506185cebdbdc11337976d2ec775eebb5

**Comment by miblanchard** [comment]
> @miblanchard , I have added react-native-gesture-handler in my forked https://github.com/flyskywhy/react-native-smooth-slider , and it works well in https://github.com/flyskywhy/react-native-slider-color-picker .
> 
> 
> 
> Maybe you can ref to the commit which just added react-native-gesture-handler :
> 
> https://github.com/flyskywhy/react-native-smooth-slider/commit/59fa00f506185cebdbdc11337976d2ec775eebb5

Thanks for sharing!  I'll take a look. I think I'm going to try and add gesture handler as an optional dependency and maintain backwards compatiblity. 

This will definitely be a good reference.  Thanks @flyskywhy
